percona/controller/pgcluster/controller.go

```diff
@@ -409,6 +410,12 @@ func (r *PGClusterReconciler) reconcileOldCACert(ctx context.Context, cr *v2.Per
 		if !k8serrors.IsNotFound(err) {
 			newCASecret.Data = oldCASecret.Data
 		}
+
+		if cr.CompareVersion("2.6.0") >= 0 && cr.Spec.Metadata != nil {
+			newCASecret.Annotations = cr.Spec.Metadata.Annotations
+			newCASecret.Labels = cr.Spec.Metadata.Labels
+		}
```

Comment:
I didn't catch this place, if we will have different annotations (for example for extentions too), should we put it to secret annotations?